composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "equip/console",
+    "description": "Equip console",
+    "type": "library",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Equip Contributors",
+            "homepage": "https://github.com/equip"
+        }
+    ],
+    "require": {
+        "equip/config": "^1.0",
+        "equip/structure": "^1.1",
+        "php": ">=5.5",
+        "symfony/console": "^3.1"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^4.8|^5.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "Equip\\Console\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "EquipTests\\Console\\": "tests/"
```

Comment:
I wouldn't use a separate namespace for this. It's way easier to use the same namespace and not have to import everything.